adam/curriculum_to_html.py

```diff
@@ -660,31 +718,12 @@ def dfs_walk(node):
                 (relation_prefix, relation_suffix) = compute_arrow(
                     relation, static_relations, first_frame_relations
                 )
-                # if matching smallerThan/biggerThan relations exist, give as single relation
-                opposite_relations = {
-                    SMALLER_THAN: BIGGER_THAN,
-                    BIGGER_THAN: SMALLER_THAN,
-                }
-                single_size_relation = None
-                if relation.relation_type in opposite_relations:
-                    if (
-                        Relation(
-                            opposite_relations[relation.relation_type],
-                            relation.second_slot,
-                            relation.first_slot,
-                        )
-                        in all_relations
-                    ):
-                        if relation.relation_type == SMALLER_THAN:
-                            single_size_relation = (
-                                f"{relation.second_slot} > {relation.first_slot}"
-                            )
-                        else:
-                            single_size_relation = (
-                                f"{relation.first_slot} > {relation.second_slot}"
-                            )
+                single_size_relation: Optional[
```

Comment:
nit: if you annotate the return type of the function you won't need the variable type annotation here.